modules/images-editing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-editing-image-registry-settings-hcp_{context}"]
+= Editing image registry settings for {hcp-title}
+
+After verifying that your image registry is enabled, you can update or edit the image registry with the `rosa edit` command.
+
+[WARNING]
```

Comment:
@chenz4027 do we want this warning in the editing module also and if so I will update it with @andreadecorte new suggetsion?

Reply:
left in both